tests/wake-format/basic/stdout

```diff
@@ -823,10 +836,8 @@ def name Unit =
 
 # floating 6
 
-
 # floating 7
 
-
 def a b c = 5
 
 def a =
```

Comment:
Yeah, why was there a double line before the middle def, but only a single one after?  Going to single-spacing hides that nicely, but if we ever want to go back to two it's going to require some digging.

Reply:
That is because floating comments (`# floating 7`) also had two trailing newlines. I just forgot to update them both to be 1